game_cfg/testing_overrides.h

```diff
@@ -0,0 +1,17 @@
+// Overrides for constants in one place
```

Comment:
I don't like this file. These are parameters for the level generation strategy. They should at least be located in that part of the code base. The level generation needs work since it should more easily accept different parameters and strategies. Until we do that I guess having something like this is alright. But not here. Put it in level.h

Reply:
I made this in response to changing parameters for testing and then changing them back before merging, and doing that each time. This is for development only. That said, I can find a different way to do this.

I wouldn't put it into any of the existing .h files because it's supposed to override anything. For example, maybe I want to change the minotaur icon to something else for testing. I only want to have one file and I would like to be able to turn all of it on and off together.

The correct way to do this, I think, would be to include a header file on the compiler command line but I got tired of trying to figure out whether it would take effect after or before the other headers.  My other idea was to include the file only if it exists, but that doesn't seem to be something the preprocessor can do. Finally, I was going to have the file only read if a def was defined on the command line via -D, but it won't compile if it references a file that doesn't exist even if it's inside a ifdef that isn't being met.

Reply:
I understand the need. Some of this will get easier when the level generation system gets expanded and abstracted. For now, keep the file yourself since it's useful but don't check it in. It's your development tool, not part of the repo.

Reply:
Yep